dis_snek/client/smart_cache.py

```diff
@@ -350,15 +344,16 @@ def get_user_guild_ids(self, user_id: "Snowflake_Type") -> List["Snowflake_Type"
     # region Message cache
 
     async def fetch_message(
-        self, channel_id: "Snowflake_Type", message_id: "Snowflake_Type", request_fallback: bool = True
+        self,
+        channel_id: "Snowflake_Type",
+        message_id: "Snowflake_Type",
     ) -> Optional[Message]:
```

Comment:
Should remove the `Optional` typing here

Reply:
How did I miss that— fixed!